airflow/models/dagrun.py

```diff
@@ -349,13 +354,16 @@ def get_previous_scheduled_dagrun(self, session: Session = None):
         ).first()
 
     @provide_session
-    def update_state(self, session: Session = None) -> List[TI]:
+    def update_state(self, session: Session = None, handle_callback: bool = True) -> List[TI]:
```

Comment:
```suggestion
    def update_state(self, session: Session = None, execute_callbacks: bool = True) -> List[TI]:
```

I think is clearer than "handle" please.

Reply:
fixed in https://github.com/astronomer/airflow/pull/913/commits/f0d64d44e82eef4cb4a5dbd85969a44de3939b99